statsmodels/regression/tests/test_lme.py

```diff
@@ -327,18 +339,19 @@ def test_vcomp_3(self):
         vc_fml = {"a": "0 + x1"}
         df = pd.DataFrame({"y": y, "x1": x1, "groups": groups})
 
-        model = MixedLM.from_formula("y ~ 1", groups="groups",
-                                     vc_formula=vc_fml,
-                                     data=df)
+        model = MixedLM.from_formula(
+            "y ~ 1", groups="groups", vc_formula=vc_fml, data=df)
         result = model.fit()
         result.summary()
 
-        assert_allclose(result.resid.iloc[0:4],
-                        np.r_[-1.180753, 0.279966, 0.578576, -0.667916],
-                        rtol=1e-3)
-        assert_allclose(result.fittedvalues.iloc[0:4],
-                        np.r_[-0.101549, 0.028613, -0.224621, -0.126295],
-                        rtol=1e-3)
+        assert_allclose(
+            result.resid.iloc[0:4],
+            np.r_[-1.180753, 0.279966, 0.578576, -0.667916],
+            rtol=1e-3)
+        assert_allclose(
+            result.fittedvalues.iloc[0:4],
+            np.r_[-0.101549, 0.028613, -0.224621, -0.126295],
+            rtol=1e-3)
```

Comment:
The older formatting here is a pattern I kinda like.  Any particular reason for disliking it?